agent.yml

```diff
@@ -47,12 +47,13 @@ api:
 #    key_file: /etc/ssl.key
 #    skip_insecure_verify: false
 
-badger:
-  mem_table_size: 1048576
-  value_log_max_entries: 1000000
-  value_log_file_size: 104857600
-  value_threshold: 1024
 agent:
+  major_ip_pattern: ".*"
+  labels:
+    env: prod
+  tags:
```

Comment:
pls comment this out, user could configure this additional metadata on purpose, but we don't do anything as default.